src/Microsoft.AspNet.HttpOverrides/HttpMethodOverrideExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.HttpOverrides;
+
+namespace Microsoft.AspNet.Builder
+{
+    public static class HttpMethodOverrideExtensions
+    {
+        /// <summary>
+        /// Allows the user to override the default method type of a request
```

Comment:
End sentences with a period.

Also, does this allow the "user" to override the method, or does it allow a "web request" to override it?